### PR TITLE
Update helm chart to 0.10.1 & Add missing variables to support AKS

### PR DESCRIPTION
**Pull request by vigohe**
- Update helm chart to use latest version `0.10.1` of Submariner
- Add missing variables `ceIPSecForceUDPEncaps, coreDNSCustomConfig, connectionHealthCheck`, that are required to set when using a Azure `AKS` kubernetes cluster

### CODE REVIEW

File: submariner-operator/questions.yml [hunk 1]
```diff
@@ -136,3 +136,9 @@ questions:
   group: "Advanced Configuration"
   description: "Cable driver implementation"
   label: "Cable Driver"
+- variable: submariner.healthcheckEnabled
+  type: boolean
+  default: true
+  group: "Advanced Configuration"
+  description: "Disable Healtcheck, if your CNI does not expose a pod IP on the nodes"
```
Comment (sridhargaddam) [inline submariner-operator/questions.yml]:
```suggestion
  description: "Disable Healthcheck"
```

As health check can be disabled for other reasons too, lets just use the term "Disable Healthcheck"

File: submariner-operator/questions.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ questions:
     type: string
     label: Submariner Operator Image Repository
   - variable: operator.image.tag
-    default: "0.7.0"
+    default: "0.10.1"
```
Comment (sridhargaddam) [inline submariner-operator/questions.yml:16]: I'm not sure if these are used during deployment
CC: @skitt @mkolesnik

**Review by sridhargaddam — COMMENTED** [review]
Thanks for pushing the PR @vigohe

File: submariner-operator/questions.yml (hunk 2 above)
Comment (vigohe) [reply submariner-operator/questions.yml:16]: https://github.com/submariner-io/submariner-charts/blob/devel/submariner-operator/templates/operator-deployment.yaml#L45

Comment (vigohe) [reply submariner-operator/questions.yml:16]: @sridhargaddam any comments?

**Comment by dfarrell07** [comment]
Thanks for the PR, sorry for being so slow to review! We clearly need more maintenance attention for Helm-based deploys, so extra-appreciated.

In CI we always override these versions to use the local operator image, which by default is the latest from `devel`. So although they default to a very old version, it's a bit better than that at least.

https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/lib/deploy_helm

I do think we need these var updates, @SteveMattar was doing something similar for the Operator bundle-based installs recently.

I'm trying to get CI to run for you here, but it currently seems like it's stuck. I may have confused it by updating the PR before giving CI permission to run. It might need a manual rebase and then CI approval?

**Comment by dfarrell07** [comment]
> I'm trying to get CI to run for you here, but it currently seems like it's stuck. I may have confused it by updating the PR before giving CI permission to run. It might need a manual rebase and then CI approval?

Ah nevermind, CI was behaving, it was just a second thing blocking it: needing the /ready-to-test label.

**Comment by SteveMattar** [comment]
The default version can be calculated according to the git tags but besides that the CRDs should be updated as well.
I'm just thinking it might be a good idea to merge the helm-charts project with submariner-operator.

**Comment by dfarrell07** [comment]
ACK @SteveMattar, there are long-term things we could maybe do to make maintenance easier. But in the meantime, let's get this one in IMHO. @skitt @sridhargaddam @tpantelis

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Review by SteveMattar — APPROVED** [review]

File: submariner-operator/questions.yml (hunk 2 above)
Comment (sridhargaddam) [reply submariner-operator/questions.yml:16]: thanks @vigohe

**Review by sridhargaddam — APPROVED** [review]

**Pull request merged**